### PR TITLE
Solana Integration: Connect and sign with phantom

### PR DESCRIPTION
**Pull request by leirbag95**
It is now possible to connect and sign message to solana network through the Phantom wallet.

The interaction with solana is done directly from a public rpc https://solana-api.projectserum.com

The address of the SPL used to calculate the balance of an account is the following:

3UCMiSnkcnkPE1pgQ5ggPCBv6dXgVUy16TmMUe1WpG9x

To know more about solana/web3js

https://solana-labs.github.io/solana-web3.js/

To know more about connection and interaction with Phantom:

https://docs.phantom.app/

User path is the same, to connect you will now have the possibility to select **Phantom** on the solana network.

connect

Your Aleph balance will be displayed as follows
balance of aleph

To sign messages, same scenario as with Metamask or another wallet, a Phantom popup will appear and will ask you to sign the message
sign message

### CODE REVIEW

**Comment by BjrInt** [comment]
Somehow version `0.5.1` was published 6 months ago (https://www.npmjs.com/package/aleph-js/v/0.5.1), even though repository was unchanged. 
This lead to an unexpected error with NPM.

Can you update your version to **0.5.3** ?

**Comment by leirbag95** [comment]
> Somehow version `0.5.1` was published 6 months ago (https://www.npmjs.com/package/aleph-js/v/0.5.1), even though repository was unchanged. This lead to an unexpected error with NPM.
> 
> Can you update your version to **0.5.3** ?

Updated ✅

File: src/store/index.js [hunk 1]
```diff
@@ -23,10 +23,11 @@ Vue.use(Vuex)
 export default new Vuex.Store({
   state: {
     erc20_address: '0x27702a26126e0B3702af63Ee09aC4d1A084EF628',
+    splToken_address: '3UCMiSnkcnkPE1pgQ5ggPCBv6dXgVUy16TmMUe1WpG9x',
     // monitor_address: '0x86bfBC59a1d1D82D2596fdeB02538fDe0426faD2', // test
     monitor_address: '0xa1B3bb7d2332383D96b7796B908fB7f7F3c2Be10',
     sender_address: '0x3a5CC6aBd06B601f4654035d125F9DD2FC992C25',
-    api_server: 'https://api1.aleph.im',
+    api_server: 'https://api2.aleph.im',
```
Comment (aliel) [inline src/store/index.js:30]: replace api2.aleph.im by official.aleph.cloud

File: src/store/index.js [hunk 2]
```diff
@@ -23,10 +23,11 @@ Vue.use(Vuex)
...
     ws_api_server: 'wss://api1.aleph.im',
```
Comment (aliel) [inline src/store/index.js:31]: official.aleph.cloud

File: src/store/index.js (hunk 1 above)
Comment (hoh) [reply src/store/index.js:30]: This is not related to this PR, I created a new issue for it: https://github.com/aleph-im/aleph-account/issues/44

File: src/store/index.js (hunk 2 above)
Comment (hoh) [reply src/store/index.js:31]: Moved to issue https://github.com/aleph-im/aleph-account/issues/44

**Comment by hoh** [comment]
Uploaded here: https://ipfs-2.aleph.im/ipfs/QmbPD6w34cUSjQtA6Kh4c99HjosjJ4DT6c1DioBpphHMCR/#/

**Pull request merged**